src/Testcontainers/Configurations/WaitStrategies/HttpWaitStrategy.cs

```diff
@@ -36,6 +36,8 @@ public sealed class HttpWaitStrategy : IWaitUntil
 
     private ushort? _portNumber;
 
+    private HttpClientHandler _httpClientHandler;
```

Comment:
Should we use the base class `HttpMessageHandler` instead?

Reply:
That would make sense, it should allow for more flexibility.

Reply:
I have adjusted the code to use the base class instead.

Reply:
Thanks 👍